### PR TITLE
added task solution

### PR DESCRIPTION
**Pull request by vlopatko**
[DEMO LINK](https://vlopatko.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,44 @@
...
+    <div className="App">
+      <h1>Dynamic list of Goods</h1>
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+      <button
+        type="button"
+        data-cy="all-button"
+        onClick={async () => setPrepareList(await getAll())}
```
Comment (anstsot) [inline src/App.tsx:20]: better to create separate functions and use `useCallback`
```suggestion
        onClick={onLoadAll}
```

**Review by anstsot — CHANGES_REQUESTED** [review]
[Checklist #1](https://github.com/mate-academy/react_dynamic-list-of-goods/blob/master/checklist.md) is not satisfied

**Review by YegorKorenkov — APPROVED** [review]
Good job!

**Comment by vlopatko** [comment]
Reopen for sync with the platform

**Review by anstsot — APPROVED** [review]

**Pull request closed without merging**